src/sql/azdata.proposed.d.ts

```diff
@@ -305,6 +306,41 @@ declare module 'azdata' {
 		required?: boolean;
 	}
 
+	/**
+	 * A property to be displayed in the PropertiesContainerComponent
+	 */
+	export interface DisplayProperty {
+		/**
+		 * The name of the property to display
+		 */
+		displayName: string;
+		/**
+		 * The value of the property to display
+		 */
+		value: string;
+	}
+
+	/**
+	 * Component to display a list of property values.
+	 */
+	export interface PropertiesContainerComponent extends Component, PropertiesContainerComponentProperties {
+
+	}
+
+	/**
+	 * Properties for configuring a PropertiesContainerComponent
+	 */
+	export interface PropertiesContainerComponentProperties {
+		/**
+		 * The properties to display
+		 */
+		displayProperties?: DisplayProperty[];
+		/**
+		 * Whether the component is currently loading
+		 */
+		loading?: boolean;
```

Comment:
I might also come back and separate out the loading part from the core component too. It's kind of weird to have this be the only one that has it built-in and I'm thinking it'd be better to just have the dashboard use the loading component separately